disnake/ui/action_row.py

```diff
@@ -236,10 +294,47 @@ def components_to_dict(components: Components) -> List[ActionRowPayload]:
 
             else:
                 raise ValueError(
-                    "components must be an Item, a list of ActionRows or a list of Items"
+                    "components must be a WrappedComponent, a list of ActionRows "
+                    "or a list of WrappedComponents"
```

Comment:
```suggestion
                    "components must be a WrappedComponent, a list of ActionRow "
                    "or a list of WrappedComponent"
```
See suggestion below